indra/literature/pmc_client.py

```diff
@@ -122,6 +122,26 @@ def extract_text(xml_string, contains=None):
     str
         Extracted plaintext.
     """
+    paragraphs = extract_paragraphs(xml_string)
+    if paragraphs:
+        return '/n'.join(paragraphs) + '/n'
```

Comment:
Why are /n-s used here? The original code uses ' '.join(paragraphs).strip(), we should preserve that unless there is a good reason not to.